hugo-example/build.gradle

```diff
@@ -7,8 +7,8 @@ buildscript {
       mavenLocal()
     }
 
-    classpath 'com.android.tools.build:gradle:1.3.0'
-    classpath 'com.jakewharton.hugo:hugo-plugin:1.2.2-SNAPSHOT'
```

Comment:
You need to install it into your local Maven repository with `./gradlew install` first. Gradle projects cannot use plugins which are build in the same project.